ntc_templates/templates/juniper_junos_show_rsvp_interface.textfsm

```diff
@@ -1,11 +1,11 @@
 Value Filldown TOTAL_INTERFACE (\S+)
 Value Required INTERFACE (\S+)
-Value STATE (\S+)
+Value OPER_STATE (\S+)
 Value ACTIVE_RESV (\S+)
 Value SUBSCRIPTION (\S+)
-Value STATIC_BW (\S+)
+Value TOTAL_BW (\S+)
 Value AVAIL_BW (\S+)
-Value RESERVED_BW (\S+)
+Value RESV_BW (\S+)
```

Comment:
When it comes to reserved bandwidth, RESERVED_BW is better in my opinion.

_This may mean we could consider a breaking change against the Alcatel RSVP template to somewhat normalize._

This is a thought and let's see what others think. Thank you!

Reply:
@jvanderaa @jmcgill298 @itdependsnetworks
Thoughts on using `RESERVED_BW` in this Juniper template and [I can] update the Alcatel template that has `RESV_BW`?

Thank you! :sweat_smile:

Reply:
To the wider group, what are the thoughts on:
`RESERVED_BW`

And similarly:
`AVAILABLE_BW`

If this is agreeable and we change this template ... then I'd work up a separate PR for the Alcatel RSVP template to align it. :relaxed:

Reply:
Thoughts?

`RESERVED_BW`
`AVAILABLE_BW`
`ACTIVE_RESERVATION`

Or do we leave things well enough alone?
(As in leave this PR as it is with the abbreviated `RESV_BW`.)